### DISCUSSION TITLE
Suppressing specific migration job feedback in console output

### DISCUSSION THREAD

**Discussion by MuriloChianfa** [laravel/framework#47730] [Ideas]
hello guys, im working with heavy migrations and i need to make a custom log for one specific migration task.

this is my migration:
![image](https://github.com/laravel/framework/assets/60560085/d9dfd30b-2a4d-4844-8d13-3a580aa1e362)

and with you can see, the migration task feedback is displayed on console output,
![laravel-migration-unwanted-output](https://github.com/laravel/framework/assets/60560085/ed68bc66-805c-4758-a69a-e5c21ceb00a2)

my ideia is to suppress this logs equal we have in seeders, calling $this->call([Seeder::class], true) -> true argument is to silence console output, but currently at migrations we not have something like these.

![image](https://github.com/laravel/framework/assets/60560085/98e9d4c4-ff6a-4f86-83a1-74c2f229533b)

with that, for this specific migration, the feedback is not shown on console,
what you guys think about that?

**Comment by yuan8** [comment]
agree
in the real world i  used two folder for this.
